redaxo/src/addons/media_manager/pages/types.php

```diff
@@ -105,6 +122,9 @@
         return $list->getColumnLink('deleteType', '<i class="rex-icon rex-icon-delete"></i> ' . rex_i18n::msg('media_manager_type_delete'));
     });
 
+    $list->addColumn('copyType', '<i class="rex-icon rex-icon-copy"></i> ' . rex_i18n::msg('media_manager_type_copy'), -1, ['', '<td class="rex-table-action">###VALUE###</td>']);
+    $list->setColumnParams('copyType', ['func' => 'copy', 'type_id' => '###id###']);
```

Comment:
In font-awsome kann man icons auch "stapeln". ggf. sollte man das icon von dem was kopiert wird mit einem anderen icon "Kombinieren" sodass man sieht dass was kopiert wird und auch gleich was kopiert wird.

siehe auch http://blog.fontawesome.io/2014/05/19/stacking-text-and-icons/